notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,7 +15,8 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    variables_will_be_in_NetCDF_file = "./STEMMUS_SCOPE/utils/csv_to_nc/Variables_will_be_in_NetCDF_file.csv"
</pre><p>I find this name a bit weird. Perhaps renaming this to, for example; <code>required_netcdf_variables.csv</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
agree. I renamed it, also fixed the naming of two columns inside the file, see [my commit](https://github.com/EcoExtreML/STEMMUS_SCOPE/pull/105/commits/bb6876cbdd7671881ea2d2131f23fba0afaf6597) in stemmus_scope repo.